app/models/enclosure_url_builder.rb

```diff
@@ -2,12 +2,32 @@
 require "addressable/template"
 
 class EnclosureUrlBuilder
+  def self.mark_authorized(enclosure_url, feed)
+    return enclosure_url unless feed.private?
+    raise "Missing tokens for private feed #{feed.id}" if feed.private? && !feed.tokens.any?
+
+    token = feed.tokens.first.token
+    add_query_param(enclosure_url, "auth", token)
+  end
+
+  # Marks the url as a `noImp`
+  # Used by Dovetail to skip the impression tracking
+  def self.mark_no_imp(enclosure_url)
+    add_query_param(enclosure_url, "noImp", "1")
```

Comment:
This is fine for now - but I feel like at some point, we'll want to not do this.  We track "bot/crawler" traffic for other apps (Spotify, Stitcher, Luminary, Adswizz, etc).  And just mark it as a bot via prx-podagent, and store the `is_duplicate = true` rows in BigQuery.